shader/main3d.frag.glsl

```diff
@@ -96,21 +96,23 @@ vec4 cc_clampValue(in vec4 value)
 
 void main()
 {
+  if (((geoMode & G_CULL_BACK) != 0) && ((geoMode & G_CULL_FRONT) != 0)) discard;
```

Comment:
This can be moved into python to ignore a material that has this setting.
While unlikely it will ever get used, it would then also skip a bunch of state changes and a drawcall alltogether

Reply:
I will remove it from the pr then